common/importer/chrome_importer_utils.cc

```diff
@@ -66,13 +99,49 @@ bool ChromeImporterCanImport(const base::FilePath& profile,
     profile.Append(base::FilePath::StringType(FILE_PATH_LITERAL("History")));
   base::FilePath passwords =
     profile.Append(base::FilePath::StringType(FILE_PATH_LITERAL("Login Data")));
-
+  base::FilePath secured_preference =
+      profile.AppendASCII(kChromeExtensionsPreferencesFile);
   if (base::PathExists(bookmarks))
     *services_supported |= importer::FAVORITES;
   if (base::PathExists(history))
     *services_supported |= importer::HISTORY;
   if (base::PathExists(passwords))
     *services_supported |= importer::PASSWORDS;
+#if BUILDFLAG(ENABLE_EXTENSIONS)
+  if (HasImportableExtensions(secured_preference))
+    *services_supported |= importer::EXTENSIONS;
+#endif
 
   return *services_supported != importer::NONE;
 }
+
+#if BUILDFLAG(ENABLE_EXTENSIONS)
+std::vector<std::string> GetImportableListFromChromeExtensionsList(
+    const base::Value& extensions_list) {
+  DCHECK(extensions_list.is_dict());
+
+  std::vector<std::string> extensions;
+  for (const auto& item : extensions_list.DictItems()) {
+    // Only import if type is extension, it's came from webstore and it's not
```

Comment:
we should also be checking to make sure it is enabled, but what about data in places like `chrome.storage` ? Are we expecting that users import extensions with all current settings or are we automatically installing whatever they had before?

Reply:
This PR targets just getting all explicitly installed extension list.
WDYT? @bsclifton @rebron

Reply:
Yup- let's do the storage as a follow up. We can create a separate issue for this 👍

Reply:
@simonhong @bridiver created https://github.com/brave/brave-browser/issues/8847 for that functionality! As-is, this is good to go 👍

Reply:
Yeah, it's good to have as a f/u :) Thanks for creating that issue.